### PR TITLE
Add mark Todo as done feature

### LINKED ISSUE #110: Add mark Todo as done feature

### PR DESCRIPTION
**Pull request by vevek**
Closes #110

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/MarkTodoCommand.java [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+            + "Parameters: PROJECT_INDEX (must be a positive integer)"
+            + PREFIX_MARK_TASK_INDEX + "TODO_INDEX \n"
+            + "Example: " + COMMAND_WORD + " 1" + " "
+            + PREFIX_MARK_TASK_INDEX + " 2";
+
+    private final Index projectIndex;
+    private final Index targetTodoIndex;
+
+    /**
+     * Creates a MarkTodoCommand to mark the specified {@code Todo} from {@code Project} as done.
```
Comment (samuelfangjw) [inline src/main/java/seedu/address/logic/commands/MarkTodoCommand.java:32]: blank line

File: src/main/java/seedu/address/logic/parser/MarkTodoCommandParser.java [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+/**
+ * Parses input arguments and creates a new MarkTodoCommand object
+ */
+public class MarkTodoCommandParser implements Parser<MarkTodoCommand> {
+
+    /**
+     * Parses the given {@code String} of arguments in the context of the MarkTodoCommand
+     * and returns a MarkTodoCommand object for execution.
+     * @throws ParseException if the user input does not conform the expected format
```
Comment (samuelfangjw) [inline src/main/java/seedu/address/logic/parser/MarkTodoCommandParser.java:22]: blank line

**Review by samuelfangjw — DISMISSED** [review]
LGTM

File: src/main/java/seedu/address/logic/parser/MarkTodoCommandParser.java (hunk 2 above)
Comment (vevek) [reply src/main/java/seedu/address/logic/parser/MarkTodoCommandParser.java:22]: Thank you. I will merge the changes and request review again :)

**Review by samuelfangjw — APPROVED** [review]
LGTM

**Pull request merged**